packages/app/pages/api/ethpass.ts

```diff
@@ -0,0 +1,54 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import type { ContractSendMethod } from "web3-eth-contract";
+import axios from "axios";
+import { getWeb3 } from "../../components/utils";
+import { getAbiFromJson } from "../../components/constants";
+import passportJson from "@cabindao/nft-passport-contracts/artifacts/contracts/Passport.sol/Passport.json";
+
+export default function handler(req: NextApiRequest, res: NextApiResponse) {
+  const { address, network, tokenId, signature, signatureMessage } =
+    typeof req.body === "string" ? JSON.parse(req.body) : req.body;
+  switch (req.method) {
+    case "POST":
+      const web3 = getWeb3(network);
+      const contract = new web3.eth.Contract(
+        getAbiFromJson(passportJson),
+        address
+      );
+      return (contract.methods.symbol() as ContractSendMethod)
+        .call()
+        .then((description) =>
+          axios.post(
+            "https://api.ethpass.xyz.com/api/v1/pass",
+            {
+              contractAddress: address,
+              tokenId,
+              signature,
+              signatureMessage,
+              pass: {
+                type: "generic",
+                description,
+              },
```

Comment:
Okay I think I'm generating the signature and signatureMessage correctly now, and getting `certificate has expired` 500 error. I think this is where we set up a certificate with Apple/EthPass? cc @christopherdro

Reply:
I'm not sure that error is coming from our service. Just checked the logs and I don't see any incoming requests. 
I'll ping you on discord.